### PR TITLE
review page feedback

### PR DESCRIPTION
**Pull request by jasalisbury**
**Description of change**
 * The url resource now shows up in activity reports review page
 * Add openapi route for the `approvers` endpoint. Held off on adding a route for submit until it is fully implemented
 * Removed old fake data from the review page (additional notes and approving managers are now blank)
 * Removed Dumbeldore as an approver as he has no "name". Added another
approver to the fake data

**How to test**

1) Pull down new changes
2) Check the `approvers` field for appropriate data
3) Fill in the `Resources used` field in `topics and resources` and make sure it shows up on the review page
4) Load up the swagger docs by browsing to `localhost:5000`

**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/135

**Checklist**

- [ ] Meets issue criteria
- [ ] Code tested
- [ ] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [ ] Documentation updated
    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified

### CODE REVIEW

File: docs/openapi/paths/activity-reports/approvers.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  description: Gets all approvers for any region the current user has write permissions
+  responses:
+    200:
+      description: All approvers for any region the user has write permissions
+      content:
+        application/json:
+          schema:
+            type: array
+            items:
+              $ref: '../../index.yaml#/components/schemas/approvingUser'
```
Comment (jasalisbury) [inline docs/openapi/paths/activity-reports/approvers.yaml:14]: I'd like for a better solution for these relative links, seems like they could potentially get pretty unwieldy

File: frontend/src/pages/ActivityReport/index.js [hunk 2]
```diff
@@ -37,8 +37,8 @@ const defaultValues = {
   topics: [],
 };
 
-const additionalNotes = 'this is an additional note';
-const approvingManagers = [2];
+const additionalNotes = '';
+const approvingManagers = [];
```
Comment (rahearn) [inline frontend/src/pages/ActivityReport/index.js:41]: Do we need these variables still? Seems like they can be removed and nothing passed on line 80?

Comment (jasalisbury) [reply frontend/src/pages/ActivityReport/index.js:41]: ahh good catch!

**Review by rahearn — APPROVED** [review]
Looks good 🎉

**Pull request merged**